MapboxNavigation/RouteMapViewController.swift

```diff
@@ -684,6 +684,15 @@ extension RouteMapViewController: NavigationViewDelegate {
             streetLabelLayer.lineOpacity = NSExpression(forConstantValue: 1)
             streetLabelLayer.lineWidth = NSExpression(forConstantValue: 20)
             streetLabelLayer.lineColor = NSExpression(forConstantValue: UIColor.white)
+
+            if ![DirectionsProfileIdentifier.walking, DirectionsProfileIdentifier.cycling].contains( router.routeProgress.routeOptions.profileIdentifier) {
```

Comment:
Btw, I think it’s possible to shorten this line based on the type of `profileIdentifier`:

```suggestion
            if ![.walking, .cycling].contains(router.routeProgress.routeOptions.profileIdentifier) {
```